src/utils/web-client.js

```diff
@@ -2,8 +2,8 @@ const Axios = require('axios');
 
 const internals = {
     //Configure this to your environment
-    host: process.env.API_HOST || 'https://qa.mybodymodel.com',
-    prefix: process.env.API_PREFIX || '',
+    host: process.env.API_HOST || 'https://staging.getcupie.com',
```

Comment:
We should probably remove the references to client projects in public areas -- my bad for putting it in here from the start.